src/widths.jl

```diff
@@ -1 +1,4 @@
\ No newline at end of file
+const WIDTHS = JSON3.read(
```

Comment:
Do we still need JSON3 here? Can't we place it as a literal array?

Reply:
It parses as the inner array as Float64 (instead of Int, Int, Float64)
But it's ok I think. Shouldn't be an issue with the current code `charCode >= x[1] && charCode <= x[2]`

```
julia> [32,32,38.67]
3-element Vector{Float64}:
 32.0
 32.0
 38.67
```

Reply:
Ok regexed it into a tuple with correct types

```
julia> Badges.WIDTHS
10035-element Vector{Tuple{Tuple{Int64, Int64}, Float64}}:
 ((32, 32), 38.67)
 ((33, 33), 43.29)
 ((34, 34), 50.49)
 ((35, 35), 90.02)
```